test/e2e/egress_test.go

```diff
@@ -335,34 +357,60 @@ func testEgressUpdateEgressIP(t *testing.T, data *TestData) {
 		originalNode     string
 		newNode          string
 		originalEgressIP string
+		oldIPRange       v1alpha2.IPRange
+		oldEgressIP      string
```

Comment:
Could you use names similar to the current names? i.e. originalIPRange, originalEgressIP.
 It seems originalEgressIP, newEgressIP were not used before.

Reply:
Done